test/config.json

```diff
@@ -35,7 +43,7 @@
                 "windowMs": 60000
             }
         }
-    },
+		},
```

Comment:
looks like we are using spaces in `test/config.json` file (as opposed to tabs like in the rest of the files). I'll update this. And maybe we should make it consistent and use tabs or spaces everywhere.